go.mod

```diff
@@ -13,6 +13,8 @@ require (
 	github.com/vantage-sh/vantage-go v0.0.14-0.20240408182501-72e700aa1c5c
 )
 
+replace github.com/vantage-sh/vantage-go v0.0.14-0.20240408182501-72e700aa1c5c => /Users/conner/vantage/vantage-go
```

Comment:
this branch is based off one that uses an unreleased version of the api codegen (https://github.com/vantage-sh/vantage-go/pull/12), and is relying on swagger doc changes to virtual tag configs beyond that.